src/index.js

```diff
@@ -1,7 +1,8 @@
 /** @module HelixUI */
+import './LICENSE.js';
 import * as Elements from './elements';
-export { version as VERSION } from '../package.json';
-export { default as Utils } from './utils';
```

Comment:
`VERSION` and `Utils` no longer exported as named exports. They are now included in the default `HelixUI` export.

Reply:
`VERSION` and `Utils` exported at the bottom of the file with everything else that's exported.